packages/manager/modules/billing/src/autoRenew/actions/terminate-all-dom/module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import angularTranslate from 'angular-translate';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ui-kit';
+import uiRouter from '@uirouter/angularjs';
+
+import component from './component';
+import routing from './routing';
+
+const moduleName = 'ovhManagerBillingAutorenewTerminateAllDom';
+
+angular
+  .module(moduleName, [
+    angularTranslate,
```

Comment:
According to the routing file, it seems that this module should have a dependency to at least:
- `ngAtInternetUiRouterPlugin`
- `ngUiRouterBreadcrumb`
Am I wrong?